lib/recurly.js

```diff
@@ -132,6 +133,7 @@ export class Recurly extends Emitter {
 
     if (options.api) {
       this.config.api = options.api;
+      this.config.origin = (new URL(options.api)).origin;
```

Comment:
Since this isn't meant to be configured, let's just compute this when it's needed by the relay.

I believe the `URL` class is not supported by Internet Explorer, which would preclude us using it. We could use a polyfill or add a util function that uses the `createElement` method.